internal/analytics/analytics.go

```diff
@@ -153,26 +156,28 @@ func EventWithLabel(category string, action string, label string) {
 }
 
 func eventWithLabel(category, action, label string) error {
-	if client == nil || condition.InTest() {
-		return nil
-	}
-	client.CustomDimensionMap(CustomDimensions.toMap())
-
-	logging.Debug("Event+label: %s, %s, %s", category, action, label)
-	return client.Send(ga.NewEvent(category, action).Label(label))
+	return sendEvent(category, action, label, CustomDimensions.toMap())
```

Comment:
This is just centralizing our send call to a single function so we don't need redundant logic for deferrals. I'd say if anything this reduces complexity. We can play more with this if and when we decide to refactor analytics (which was not the purpose of this story).

Reply:
I marked this at this line, not because of the function call itself, but because the diff shows a loss of the InTest check at this line. Again, just concerned about what not checking InTest will do during artifact test runs.